src/ex1/script.js

```diff
@@ -0,0 +1,82 @@
+let pendingTaskCount = 0;
+
+const taskInput = document.getElementById("todoTaskInput");
+const clearButton = document.getElementById("clearButtonId");
+
+taskInput.addEventListener("keypress", function(event) {
+    if (event.key === "Enter") {
+        event.preventDefault();
+        document.getElementById("addTaskButtonId").click();
+    }});
+
+if(pendingTaskCount === 0){
+    startingPInsert();
+    displyPendingTasks();
+}
+
+function startingPInsert(){
+    const startingP = document.createElement("p");
+    startingP.classList.add("startingP");
+    startingP.innerHTML = "Start To Arrange Your Tasks!";
```

Comment:
Why initialize this every time? think about it, this sentence stays the same. it's just about visibility.